### PR TITLE
add result description for algorithm

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/graph-computing/algorithm-description.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ Nebula Graph supports some graph computing tools. This topic describes the algor
     Different graph computing tools support different algorithms and different parameters. See below for details.
 -->
 
+!!! note
+
+    The algorithm parameters need to be set when performing graph computing, and the data source has requirements. The data source needs to contain starting vertexes and destination vertexes. PageRank, DegreeWithTime, SSSP, APSP, LPA, HANP, and Louvain algorithms must include weight.
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:21]: and the data source has requirements ---> and there are requirements for data sources

Comment (abby-cyber) [reply docs-2.0/graph-computing/algorithm-description.md:21]: vertexes--->vertices
starting vertexes ---> source vertices

File: docs-2.0/graph-computing/algorithm-description.md [hunk 2]
```diff
@@ -35,12 +43,21 @@ Parameter descriptions are as follows:
...
+    |`IS_DIRECTED`|`true`| Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
+    |`EPS`|`0.0001`| The convergence accuracy. When the difference between the result of two iterations is less than the `EPS` value, the iteration is not continued.|
+    |`DAMPING`|`0.85`| The damping coefficient. It is the jump probability after visiting a page.|
+
+  - Output parameters
+
+    |Parameter|Type|Description|
+    |:--|:--|:--|
+    |`VID`|Determined by `vid_type` | The vertex ID.|
+    |`VALUE`|double| The PageRank value of the vertex.|
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:60]: vertex or vertices?

File: docs-2.0/graph-computing/algorithm-description.md [hunk 3]
```diff
@@ -59,14 +76,27 @@ Parameter descriptions are as follows:
...
-  |`EDGES`|`0`| The number of edges. If set to `0`, the system automatically calculates the value.|
-  |`KMIN`|`1`| Set the minimum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`. |
-  |`KMAX`|`1000000`| Set the maximum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`.|
-  |`ITERATIONS`|`10`| The maximum number of iterations.|
+  - Input parameters
+
+    |Parameter|Predefined value|Description|
+    |:--|:--|:--|
+    |`TYPE`|`vertex`| The calculation type. Available values are `vertex` and `subgraph`. When set to `vertex`, the system calculates the number of cores for each vertex.|
+    |`KMIN`|`1`| Set the minimum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`. |
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:84]: what do you mean by when range calculation? does it refer to when calculating the range?

File: docs-2.0/graph-computing/algorithm-description.md [hunk 4]
```diff
@@ -59,14 +76,27 @@ Parameter descriptions are as follows:
...
+    |`KMAX`|`1000000`| Set the maximum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`.|
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:85]: ditto

File: docs-2.0/graph-computing/algorithm-description.md [hunk 5]
```diff
@@ -59,14 +76,27 @@ Parameter descriptions are as follows:
...
+
+  - Output parameters when `TYPE=vertex`
+
+    |Parameter|Type|Description|
+    |:--|:--|:--|
+    |`VID`|Determined by `vid_type`| The vertex ID.|
+    |`VALUE`|int| Outputs the core degree of the vertex.|
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:92]: the vertex -->vertices or one vertex  


the following content have the same issue

File: docs-2.0/graph-computing/algorithm-description.md [hunk 6]
```diff
@@ -175,9 +272,13 @@ The APSP (Full Graph Shortest Path) algorithm is used to find all shortest paths
...
 Parameter descriptions are as follows:
 
-|Parameter|Predefined value|Description|
-|:--|:--|:--|
-|`WEIGHT`|-| The maximum weight of edges.|
+- Output parameters
+
+  |Parameter|Type|Description|
+  |:--|:--|:--|
+  |`VID1`|Determined by `vid_type`| The VID of the starting vertex.|
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:279]: ditto

please check the same issue in the following content

File: docs-2.0/graph-computing/algorithm-description.md [hunk 7]
```diff
@@ -275,11 +392,20 @@ Parameter descriptions are as follows:
...
+    |`ITERATIONS`|`10`|The maximum number of iterations.|
+    |`IS_DIRECTED`|`true`|Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
+    |`IS_CALC_MODULARITY`|`false`| Whether to calculate modularity.|
+
+  - Output parameters
+
+    |Parameter|Type|Description|
+    |:--|:--|:--|
+    |`VID`|Determined by `vid_type`| The vertex ID.|
+    |`LABEL`|The same type as `VID`| Outputs the vertex IDs that are the same as the label.|
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:408]: what do you mean "the same type as VID"   

the same data type as that of the VID?

Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:408]: Outputs the vertex IDs that are the same as the label. --->Outputs the vertex IDs that have the same label.

File: docs-2.0/graph-computing/algorithm-description.md [hunk 8]
```diff
@@ -299,12 +425,21 @@ Parameter descriptions are as follows:
...
+    |`IS_DIRECTED`|`true`|Whether to consider the direction of the edges. If set to `false`, the system automatically adds the reverse edge.|
+    |`PREFERENCE`|`1.0`| The bias of the neighbor vertex degree. `m>0`indicates biasing the neighbor with high vertex degree, `m<0` indicates biasing the neighbor with low vertex degree, and `m=0` indicates ignoring the neighbor vertex degree.|
+    |`HOP_ATT`|`0.1`|The attenuation coefficient. The value ranges from `0` to `1`. The larger the value, the faster it decays and the fewer times it can be passed.|
+
+  - Output parameters
+
+    |Parameter|Type|Description|
+    |:--|:--|:--|
+    |`VID`|Determined by `vid_type`| The vertex ID.|
+    |`LABEL`|The same type as `VID`| Outputs the vertex IDs that are the same as the label.|
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/algorithm-description.md:442]: ditto 
please check the following content

File: docs-2.0/graph-computing/algorithm-description.md (hunk 2 above)
Comment (cooper-lzy) [reply docs-2.0/graph-computing/algorithm-description.md:60]: Vertex. There is only one value in a row.

**Review by whitewum — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Pull request merged**